src/dc_receive_imf.rs

```diff
@@ -32,17 +32,28 @@ enum CreateEvent {
     IncomingMsg,
 }
 
+pub async fn dc_receive_imf(
```

Comment:
Maybe remove this wrapper and rename `dc_receive_imf_inner` into `dc_receive_imf`?

Reply:
Problem is, I will have to add an extra `false` parameter to 20-30 tests then (it's a pity that Rust does not have default parameters). 

I could rename this wrapper to `dc_receive_imf_test` and maybe move it to `test_utils.rs` and then rename `dc_receive_imf_inner` into `dc_receive_imf`.